lib/stellar/account_info.rb

```diff
@@ -0,0 +1,30 @@
+require 'toml-rb'
+require 'uri'
+
+module Stellar
+  class AccountInfo
```

Comment:
I don't think the introduction of this class is super useful at this point.  Keeping this up to date will be a pain and it doesn't give much benefit.